backend/siarnaq/api/compete/models.py

```diff
@@ -163,7 +163,7 @@ def for_saturn(self):
 
     def enqueue_options(self):
         """Return the options to be submitted to the compilation queue."""
-        report_url = "https://{}{}".format(
+        report_url = "http://{}{}".format(
```

Comment:
Why `http` rather than `https`?

Reply:
Staging doesn't run HTTPS so it fails there. Production runs HTTP–>HTTPS redirect so this change shouldn't break there.